### ISSUE TITLE
Training parameters for inception v4 and Inception-resnet-v1/v2

### ISSUE THREAD

**Issue by thadpasce16** [tensorflow/models#876]
## models/slim/nets/inception_v4.py

I have a question about the training hyper-parameters for inception_v4 and inception_resnet_v1/2 models.

Anybody here has successfully trained inception_v4 and inception_resnet variants (i.e. say got 80.2% top-1 accuracy on Inception_v4?)? What training hyperparameters are you using? Here is what I am using but I am not getting accuracy mentioned here: [https://github.com/tensorflow/models/tree/master/slim#pre-trained-models](url) 

Note that the rmsprop and other parameters I specify is my best effort read of what the Inception v4 paper says.

# Run training.
python train_image_classifier.py \
  --num_clones=4 \
  --train_dir=${TRAIN_DIR} \
  --dataset_name=imagenet \
  --dataset_split_name=train \
  --dataset_dir=${DATASET_DIR} \
  --model_name=inception_v4 \
  --preprocessing_name=inception_v4 \
  --max_number_of_steps=1000000 \
  --batch_size=128 \
  --save_interval_secs=600 \
  --save_summaries_secs=600 \
  --log_every_n_steps=100 \
  --optimizer=rmsprop \
  --rmsprop_decay=0.9 \
  --learning_rate=0.045 \
  --learning_rate_decay_factor=0.94 \
  --num_epochs_per_decay=2

# Run evaluation.
python eval_image_classifier.py \
  --checkpoint_path=${TRAIN_DIR} \
  --eval_dir=${TRAIN_DIR} \
  --dataset_name=imagenet \
  --dataset_split_name=validation \
  --dataset_dir=${DATASET_DIR} \
  --model_name=inception_v4

If you have gotten the accuracies mentioned in the above link, could you share your hyper-params configs? Thanks

**Comment by jart** [comment]
Thanks for reaching out. We try to keep this issue tracker limited to bugs and feature requests. We recommend posting on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) for community-driven support.

**Comment by wenwei202** [comment]
Anywhere we can find the hyper-parameters for training other models (vgg, resnet, etc) instead of the default inception model? Only in papers? Thanks.
